.nvmrc

```diff
@@ -0,0 +1 @@
+16.13
```

Comment:
Okay no problem. The only reason I had added this was because I had issues running the project when I started because I was using Node 17.4. I wasn't sure if there was any benefit to standardizing on a version.